radio/src/dataconstants.h

```diff
@@ -255,6 +255,7 @@ enum UartModes {
   UART_MODE_NONE,
 #endif
   UART_MODE_TELEMETRY_MIRROR,
+  UART_MODE_FLYSKY_GIMBAL,
```

Comment:
It could be added after UART_MODE_LUA, so it will not change option for users with enabled UART_MODE_SBUS_TRAINER.

Reply:
Removed as discussed